job/server/src/main/java/alluxio/job/plan/stress/StressBenchDefinition.java

```diff
@@ -91,23 +97,56 @@ public StressBenchDefinition() {
     return result;
   }
 
+  private Map<String, String> getUfsConf(String ufsUri, RunTaskContext runTaskContext)
+      throws Exception {
+    Map<String, MountPointInfo> mountTable = runTaskContext.getFileSystem().getMountTable();
+    for (Map.Entry<String, MountPointInfo> entry : mountTable.entrySet()) {
+      if (PathUtils.hasPrefix(ufsUri, entry.getKey())) {
```

Comment:
Will this work with nested mounts?

Reply:
yes, this is a ufs Uri. We can't mount both a parent Ufs URI and a sub directory ufs URI.